### DISCUSSION TITLE
Welcome to Potion-Effects Discussions!

### DISCUSSION THREAD

**Discussion by Tellinq** [Tellinq/Potion-Effects#1] [Announcements]
## 👋 Welcome!
  Welcome to Discussions! We use Discussions to allow people to make suggestions to further improve this mod.

Want to suggestion something or start a discussion? Make a post!

**Comment by he3als** [comment]
okay

**Reply by Tellinq** [reply]
ok

**Reply by he3als** [reply]
ok
